tests/performance/load/groups/alerts.js

```diff
@@ -0,0 +1,40 @@
+import {Client, StatusOK} from 'k6/net/grpc';
+import {check, group} from 'k6';
+
+const GRPC_PORT = ":443";
+
+const client = new Client();
+
+client.load(['../../../proto', '../../../third_party/googleapis'], 'api/v1/alert_service.proto');
+
+export function alertsGrpc(host, headers, tags) {
+
+    host = host.replace('https://', '')
+    host = host.includes(':') ? host : host + GRPC_PORT;
+    client.connect(host);
+
+    group('list alerts grpc', function () {
+    [0, 10, 100, 1000].forEach(limit => {
+        tags.limit = limit;
+        const params = {
+            metadata: headers, tags: tags,
+        };
+        const response = client.invoke(
+            'v1.AlertService/ListAlerts',
+            {
+                pagination: {
+                    limit: limit,
```

Comment:
We are testing with different limits, but how many alerts are actually returned? Is there a way for the test to output that information?

Reply:
We can add this number to tags in check